src/App.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import './App.css';
+import { GoodsList } from './components';
+
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+class App extends React.Component {
+  state = {
+    goods: [],
+    isReversed: false,
+    sortBy: '',
+  };
+
+  start = () => {
+    this.setState({
+      goods: [...goodsFromServer],
```

Comment:
do you need goods in the state?

Reply:
I thought yes cuz it's analog of getting goods from the server and setting them to the state (Misha recommended thinking this way).
But I change, so check pls